### DISCUSSION TITLE
Where can I find the various methods usable in this package?

### DISCUSSION THREAD

**Discussion by TechPro424** [backtrackapp/spotify.js#16] [Q&A] [answered]
Where can I find the various methods usable in this package? I know there's going to be a documentation soon, but where can I find the methods as of now?

**Comment by stijnvdkolk** [comment]
Look through the code or use the intellisense provided with the typings.

**Comment by stingalleman** [comment, marked as answer]
The docs are now located at https://spotistatsapp.github.io/spotify.js
